src/app/lib/utilities/preprocessor/rules/topstories.js

```diff
@@ -21,7 +21,7 @@ const squashTopStories = jsonRaw => {
       if (squashKeys.includes(group.type)) {
         collectedItems = collectedItems.concat(group.items);
 
-        const foundStapline = pathOr(null, ['strapline', 'name'], group);
+        const foundStapline = pathOr(null, ['title'], group);
```

Comment:
This is temporary to allow us test designs for the `top-stories` without a `strapline`. Most `top-stories` in the test data lack the `strapline`. Adding `strapline` data to the data would be time consuming.